services/usb-livesync-service-base.ts

```diff
@@ -39,7 +39,7 @@ class SyncBatch {
 					});
 				}
 				this.timer = null;
-			}, 500);
+			}, 250);
```

Comment:
why 250, not 125 ? comment or constant could be helpful for the next time we check this code